package.json

```diff
@@ -127,6 +158,7 @@
           "scope": "window"
         },
         "sonarlint.connectedMode.servers": {
+          "deprecationMessage": "The setting is deprecated. Use `sonarlint.connectedMode.connections.sonarqube` or `sonarlint.connectedMode.connections.sonarcloud` instead.",
```

Comment:
Nitpick: looks like markdown is not supported here (yet?)

Reply:
I saw that, but backtips are not rendering too bad, and this is future proof. Or do you prefer to use  single quotes?

Reply:
Nope, fine by me!